src/components/tasks/card/suggestion.module.scss

```diff
@@ -0,0 +1,64 @@
+.suggestions {
+    border: 1px solid #999;
+    list-style: none;
+    margin-top: 0;
+    max-height: 10rem;
+    padding-left: 0;
+    max-width: 12rem;
+    min-width: 7rem;
+    overflow: auto;
+    background-color: white;
+    position: absolute;
+    top: 20px;
+    left: -120px;
+    border-radius: 5px;
+    z-index: 100;
+    li {
+        padding: 0.5rem;
```

Comment:
try to use one type of unit, if using rem please stick to rem wherever possible

Reply:
rem, as i believe will scale up or down depending on user's device. So it makes sense using it for margin, padding or size.
But here i do want this box to appear 20px from top and -120px from left, relative to the input box, regardless of user's device. So therefore I used an absolute unit like px. Please correct me, if my thinking was wrong.

Reply:
does make sense to what you said also @fakhruddinkw please check on responsiveness that this doesnt break anything

Reply:
I have tested this on different screens, It works @vvaibhavdesai

Reply:
you are correct on this @fakhruddinkw its not a rule to stick with one unit, we need to understand which to use where